### PR TITLE
Preview ui

### PR DESCRIPTION
**Pull request by TimYuenior**
You can choose your shot with the choicebox in the top left.
You can adjust the estimated duration per shot.
Pressing play uses these adjusted durations per shot.

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/gui/PreviewController.java [hunk 1]
```diff
@@ -40,133 +42,220 @@
...
+    @FXML private Label error;
+
+    private SimpleIntegerProperty currentShot;
+    private int currentSelected;
     private Timeline timeline;
     private ArrayList<Shot> shots;
     private ArrayList<ImageView> views;
 
     @FXML
     private void initialize() {
```
Comment (tkolenbrander) [inline src/main/java/nl/tudelft/contextproject/gui/PreviewController.java:61]: This method is too long. Maybe you can split it up in 'Dummy settings' and 'Listeners'?

**Comment by tkolenbrander** [comment]
Travis fails, take a look at the .travis.yml and pom.xml of Steven in the 'gui_script' branch on how to fix it.

**Comment by stevenmeijer** [comment]
I have no additional comments, fix @tkolenbrander's issues and then you are good to go.

**Comment by tkolenbrander** [comment]
Travis passes now, great! Just resolve the merge conflicts and I will accept the pull request

**Comment by tkolenbrander** [comment]
It takes a really long time before travis starts, what is up with that?

**Comment by TimYuenior** [comment]
I have no clue, I only took the pom from the gui_script branch to resolve the conflicts.

**Comment by tkolenbrander** [comment]
The server were probably busy. Anyway, travis passes, merge conflicts are resvoled. Let's merge!

**Pull request merged**